neuro_san/internals/chat/data_driven_chat_session.py

```diff
@@ -76,6 +80,8 @@ def __init__(self, agent_network: AgentNetwork):
 
         self.front_man: FrontMan = None
         self.sly_data: Dict[str, Any] = {}
+        self.invocation_context: InvocationContext = None
+        self.interceptor: InterceptingJournal = None
```

Comment:
These guys now have to be state in order to have the run_it() function correctly at this level.